helsingborg-stad/helsingborg-io-sls-api
# Feat/create case contacts

## Explain the changes you’ve made

Read default case contacts from S3 when creating cases.

## Explain why these changes are made

New case structure for managing contacts on a case.

## How to test

Concrete example:

Use [resources PR 130](https://github.com/helsingborg-stad/helsingborg-io-sls-resources/pull/130).

Add a `contacts.json` file in the root of the new config bucket. Sample contents:

```json
[
  {
    "name": "KC",
    "description": "Hej! Ring [Kontaktcenter](tel://0767000000)"
  }
]
```

1. Checkout this branch
2. Create a new viva case via the API (e.g. simulate login for a valid user)
3. Verify contacts are added to the case
